src/quickgui/plugin/qgsquickphotopanel.qml

```diff
@@ -38,12 +38,21 @@ Drawer {
   property var captureButtonIcon: QgsQuick.Utils.getThemeIcon("ic_camera_alt_border")
   property var confirmButtonIcon: QgsQuick.Utils.getThemeIcon("ic_check_black")
   property var cancelButtonIcon: QgsQuick.Utils.getThemeIcon("ic_clear_black")
+  property var backButtonSource: QgsQuick.Utils.getThemeIcon("ic_back")
   property real imageButtonSize: 45 * QgsQuick.Utils.dp
   property real buttonSize: imageButtonSize * 1.2
   property var buttonsPosition
 
   signal confirmButtonClicked(string path, string filename)
 
+  function cancelEvent() {
```

Comment:
can we rename this method - e.g. discardCapturedImage() would make it more clear that we're not canceling the whole panel, only the image we already took...